ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1410,7 +1410,7 @@
   {
     "id": "python-default",
     "creator": "ide",
-    "name": "Python",
+    "name": "Python default",
```

Comment:
"Python default" is "Default Python Stack with Python 3.5.1, pip 8.1.1."
and
new "Python" is "Default stack with Python 3.7"

Looks like they both are default =)
Maybe name can contain info Python version, or other software that is installed there.

note tha escription mentions pip 8.1.1 and component contains 10.0.1

Reply:
or maybe the actual difference is that one is che 6 and another che7 based?

Reply:
yes, the first one is for Che6 and another one is for Che7.
The stack for Che6 will be removed eventually.

Reply:
@sleshchenko yes I asked @tolusha to do some minor changes to the PR considering that this stack will eventually replace the Che 6 one and to be more consistent with the Che 7 Java stack.